docs/app/js/codepen.js

```diff
@@ -0,0 +1,175 @@
+(function() {
+  DocsApp
+    .factory('$codepenDataAdapter', CodepenDataAdapter)
+    .factory('$codepen', ['$demoAngularScripts', '$document', '$codepenDataAdapter', Codepen]);
+
+  /**
+   * @ngdoc service
```

Comment:
Does docs code here need all this `@ngdoc` stuff? I don't think we would be generating documentation from it. 

Here, for example, you should be able to just do

```js
/**
 * Service for generating CodePens from demo-site examples.
 * @constructor @ngInject
 */
function Codepen(...) {

}
```